.travis.yml

```diff
@@ -59,7 +59,7 @@ after_success:
       ssh-keyscan -t ed25519 "${APP_DEPLOY_HOST}" >> ~/.ssh/known_hosts
 
       docker tag usedgoodreads/app usedgoodreads/app:staging
-      docker image save usedgoodreads/app:staging | ssh "${APP_DEPLOY_USER}@${APP_DEPLOY_HOST}" "docker image load"
+      docker image save usedgoodreads/app:staging | ssh "${APP_DEPLOY_USER}@${APP_DEPLOY_HOST}" "docker system prune -f -a ; docker image load"
```

Comment:
What happens if the cleanup fails for some reason? Then the new image will not be loaded, right? 

And is there an edge case that the container is not running while this is being executed? If that happens and `image load` fails for some reason there are no images of anything on the machine

Reply:
> What happens if the cleanup fails for some reason? Then the new image will not be loaded, right?

Wrong :P That's exactly the reason we are doing `cmd1 ; cmd2` and not `cmd1 && cmd2`.

And yes in the case nothing is running _and_ image load fails for some reason (why?) then we're out of luck and e.g. have to manually deploy or make an (potentialy empty) commit to kick off another deploy.

Do you think there's a better way and a safer way to do this? :thinking:

Reply:
No. Seems valid. Loading could fail if there is no connection to the machine. But it is a pretty rare case and an empty commit could help. No worries